packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -20,18 +23,41 @@ export function Service({
   location,
   lastAccessTimeFormatted,
   canSignOut,
+  isCurrentSession,
+  clientId,
+  deviceId,
+  refreshTokenId,
+  sessionTokenId,
 }: {
   name: string;
   deviceType: string | null;
+  deviceId: string | null;
   location: DeviceLocation;
   lastAccessTimeFormatted: string;
   canSignOut: boolean;
+  isCurrentSession: boolean;
+  clientId: string;
+  sessionTokenId: string | null;
+  refreshTokenId: string | null;
 }) {
   const alertBar = useAlertBar();
   const { city, stateCode, country } = location;
   const locationProvided = Boolean(city && stateCode && country);
   let serviceLink, iconName;
   const [modalRevealed, revealModal, hideModal] = useBooleanState();
+  const [
+    secondaryModalRevealed,
+    revealSecondaryModal,
+    hideSecondaryModal,
+  ] = useBooleanState();
+  const [disconnectReason, setDisconnectReason] = useState<string>();
+  const [disconnected, setDisconnected] = useBooleanState(false);
+  // Note: setting a default start time to avoid TS complaints about submitStartTime
+  // possibly being undefined. It's not a meaningful number.
+  const [submitStartTime, setSubmitStartTime] = useState<number>(Date.now());
```

Comment:
We could add a `!` to where the error is (`submitStartTime!`) to let the type system that we know what we are doing.  Or, if we need to initialize a value to satisfying the type system, then let's use an obviously incorrect value like `0`?